frontend/src/components/user/profile/ProfileRelated.tsx

```diff
@@ -17,10 +17,15 @@ const mapToUserAvatar = (userList) =>
     />
   ));
 
+const hasMinesParent = (profile: Profile) => (profile.minesparent && profile.minesparent.length > 0);
+const hasFillot = (profile: Profile) => (profile.fillots && profile.fillots.length > 0);
+const hasCousinAst = (profile: Profile) => (profile.cousinast && profile.cousinast.length > 0);
+
 export const ProfileRelated = ({ profile }: { profile: Profile }) =>
-  profile.roommate.length > 0 ||
-  profile.minesparent.length > 0 ||
-  profile.fillots.length > 0 ? (
+    profile.roommate.length > 0 ||
```

Comment:
Maybe we could create a `hasRoomate` function to be consistent with `fillot`etc...